### PR TITLE
[zh-cn]: Update translation of Function.displayName

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#15403

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/function/displayname/index.md [hunk 1]
```diff
@@ -5,78 +5,92 @@ slug: Web/JavaScript/Reference/Global_Objects/Function/displayName
...
+testName("foo<"); // function foo()
+testName("foo..."); // function foo()
+testName("foo(^)"); // function foo()
 ```
 
-## Specifications 规范
+## 规范
 
-不属于任何规范
+不属于任何规范.
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/function/displayname/index.md:88]:
```suggestion
不属于任何规范。
```

File: files/zh-cn/web/javascript/reference/global_objects/function/displayname/index.md [hunk 2]
```diff
@@ -5,78 +5,92 @@ slug: Web/JavaScript/Reference/Global_Objects/Function/displayName
...
-function doSomething() {}
+## 描述
 
-console.log(doSomething.displayName); // "undefined"
+如果存在 `displayName` 属性，其可能会被控制台和性能分析工具优先选择，作为函数的名称显示。
 
-var popup = function (content) {
-  console.log(content);
-};
+在各种浏览器中，只有 Firefox 控制台使用该属性。React 开发工具在显示组件树时也使用 [`displayName`](https://reactjs.org/docs/higher-order-components.html#convention-wrap-the-display-name-for-easy-debugging) 属性。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/function/displayname/index.md:18]:
```suggestion
在各种浏览器中，只有 Firefox 控制台使用该属性。React 开发者工具在显示组件树时也使用 [`displayName`](https://reactjs.org/docs/higher-order-components.html#convention-wrap-the-display-name-for-easy-debugging) 属性。
```

File: files/zh-cn/web/javascript/reference/global_objects/function/displayname/index.md [hunk 3]
```diff
@@ -5,78 +5,92 @@ slug: Web/JavaScript/Reference/Global_Objects/Function/displayName
...
 
-popup.displayName = "Show Popup";
+Firefox 对可能由[匿名 JavaScript 函数命名约定算法](http://johnjbarton.github.io/nonymous/index.html)生成的 `displayName` 进行了一些基本的解码尝试。以下模式可以被检测到：
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/function/displayname/index.md:20]:
```suggestion
Firefox 对可能由[匿名 JavaScript 函数命名约定](http://johnjbarton.github.io/nonymous/index.html)算法生成的 `displayName` 进行了一些基本的解码尝试。以下模式可以被检测到：
```

File: files/zh-cn/web/javascript/reference/global_objects/function/displayname/index.md [hunk 4]
```diff
@@ -5,78 +5,92 @@ slug: Web/JavaScript/Reference/Global_Objects/Function/displayName
...
 console.log(object.someMethod.displayName); // "someMethod (123)"
 ```
 
-## Examples 例子
+### 清理 displayName
 
-这个显示名称通常在控制台和配置文件中，用它来提醒对它背后的真实函数名 {{jsxref("Function.name", "func.name")}}的引用。例如：
-
-通过如下的举例，显示的名称应该显示像"function My Function()"
+Firefox devtools 会在显示 `displayName` 属性之前清理一些常见的模式。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/function/displayname/index.md:62]:
```suggestion
Firefox 开发者工具会在显示 `displayName` 属性之前清理一些常见的模式。
```

**Review by yin1999 — COMMENTED** [review]
Great. Just some suggestions

**Review by yin1999 — APPROVED** [review]
Thank you @JasonLamv-t

**Pull request merged**